app/scripts/lib/fxa-client.js

```diff
@@ -478,52 +443,43 @@ define(function (require, exports, module) {
      * @param {Relier} relier
      * @returns {Promise} resolves with new session information on success.
      */
-    changePassword: function (originalEmail, oldPassword, newPassword, sessionToken, sessionTokenContext, relier) {
+    changePassword: withClient((client, originalEmail, oldPassword, newPassword, sessionToken, sessionTokenContext, relier) => {
```

Comment:
I was unsure how to handle that, depends on the consumer of the API docs. A consumer of the API doesn't pass in `client` and doesn't need to know about it. A developer modifying the function does. I chose the POV of a consumer of the API.